### PR TITLE
[HR] Part-time ratio retroactive adjustments

### PR DESCRIPTION
**Pull request by cassus**
I told this to potential part-time partners, also heard from Petak telling this. So let's make it official.
This would also solve some of the tensions around Banyo's last 3 months.

Otherwise make it explicit that there are no retroactive adjustments and make sure part-time people don't put in significantly more resources than committed.

### CODE REVIEW

**Comment by cassus** [comment]
@peterlangmar pls check as Biz role cross-link .

File: HR.md [hunk 1]
```diff
@@ -55,8 +55,15 @@ Probation time is 3 months, when it comes to ownership there is a one year cliff
 Salary formula, levels, Glue levels and Glue multiplier: https://docs.google.com/spreadsheets/d/1voxCpH29Lr8OlSuG7JL51BQSAwQPfVzkdpkM7aR9GbU/edit#gid=118044564
 
 ##### Policy: Part-time
-If you are an x% part-time employee in the Lab, you get x% pay and x% ownership.
-This is a company level policy.
+- If you are an `x%` part-time employee in the Lab, you get `x%` pay and `x%` ownership.
+- You can't go above 100%
+
+##### Policy: Part-time ratio retroactive adjustments
+If your commitment was `x%` part-time but you spent significantly more `y%` of your efforts on the Lab you can retroactively adjust your part-time ratio.
```
Comment (peterlangmar) [inline HR.md]: I have an issue with this due 'you can't go over 100%'. Overtime in an early stage company and as a shareholder is normal.

File: HR.md [hunk 2]
```diff
@@ -55,8 +55,15 @@ Probation time is 3 months, when it comes to ownership there is a one year cliff
...
+
+- With the explicit support of the two people you most worked with.
+- Not before the last divident pay.
```
Comment (peterlangmar) [inline HR.md]: So the time increase would only go to action once paid?

**Comment by peterlangmar** [comment]
I like it in general

File: HR.md (hunk 1 above)
Comment (cassus) [inline HR.md]: It's okay to work more than 100%. But that's still counted as full time. Not tracking or paying for overtime as before.

I'm not sure if I understand you. What is the tension this proposal created?

File: HR.md (hunk 2 above)
Comment (cassus) [inline HR.md]: Clarified in the proposal

File: HR.md (hunk 1 above)
Comment (peterlangmar) [inline HR.md]: So we agree now that I'm working 50%, but I work 60% than you pay me 10% more and give me a high glue. But if we agree that I work 100% and I work 110% than you maybe increase my glue. I dont get this as this prefers part time people over full time people

**Comment by peterlangmar** [comment]
ok, I wont have objections against this proposal as it is still better to 'break' glue + increase time than to increase seniority to pay more

**Comment by EggDice** [comment]
:+1:

**Comment by cassus** [comment]
@EggDice please merge as 12 days have passed.

See https://github.com/lab-coop/lab-governance/blob/master/README.md#policy-async-governance-process-tlc

**Pull request merged**